pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -1,7 +1,7 @@
 ---
-title: <%= I18n.t('dashboard_landing_title') %>
 theme: angular
 angular: teacherDashboard
+title: <%= I18n.t('dashboard_landing_title').inspect %>
```

Comment:
It returns a string, but it's possible that the string has a character that would need to be quoted/escaped in yaml (in particular, if there's no loc string for that language it will say "translation missing: dashboard_landing_title...".

> On Nov 24, 2015, at 8:27, Andrew Oberhardt <notifications@github.com> wrote:
> 
> In pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml:
> 
> >  theme: angular
> >  angular: teacherDashboard
> > +title: <%= I18n.t('dashboard_landing_title').inspect %>
> Is inspect needed here? Doesn't I18n.t already return a string?
> 
> —
> Reply to this email directly or view it on GitHub.
>